misc/python/materialize/cli/mzcompose.py

```diff
@@ -424,6 +425,8 @@ def run(self, args: argparse.Namespace) -> None:
                 # coverage directory as the current user, so Docker doesn't create
                 # it as root.
                 (composition.path / "coverage").mkdir(exist_ok=True)
+                # Need materialize user to be able to write to coverage
+                os.chmod(composition.path / "coverage", 0o777)
```

Comment:
Weird, apparently the default mode in `os.mkdir(path, mode=0o777, *, dir_fd=None)[¶](https://docs.python.org/3/library/os.html#os.mkdir)` is being ignored.

Reply:
Strange indeed, we are using pathlib, but it has the same default: https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir
And as far as I know we are not creating the directory elsewhere.